src/laws/Laws.jl

```diff
@@ -82,8 +82,11 @@ U_law = LawU(nn_model, params; max_NN = 50.0, prescale_bounds = [bounds_H, bound
 function LawU(
     nn_model::NeuralNetwork,
     params::Sleipnir.Parameters;
-    max_NN::Union{F, Nothing} = 50.0,
-    prescale_bounds::Union{Vector{Tuple{F,F}}, Nothing} = [(0.0, 300.0), (0.0, 0.5)],
+    max_NN::Union{F, Nothing} = nothing,
+    prescale_bounds::Union{Vector{Tuple{F,F}}, Nothing} = nothing,
+    precompute_interpolation::Bool = true,
+    precompute_VJPs::Bool = true,
+    dummy_float::F = 1.0, # This is just to recognize the type F when no float argument is provided. Not always, but sometimes this produces an error of "F not been defined"... I am sure there is a better solution to this but I couldn't find it. Doing this for now.
```

Comment:
@albangossard @JordiBolibar do you have any suggested solution for this? When I pass the possible `F` arguments as `nothing` (in this case, `prescale_bounds` and `max_NN`) I get sometimes (but not always, which is weird and annoying) an error since the type `F` cannot be found by the arguments (of course, since there are no float). Do you know how to fix this? I just introduced a dummy float variable so `F` is defined by default, just to be able to run this code for now.